server/task/task-distributed/src/main/java/org/apache/james/task/eventsourcing/distributed/RabbitMQWorkQueue.java

```diff
@@ -123,7 +124,7 @@ void declareQueue() {
         Mono<AMQP.Queue.DeclareOk> declareQueue = sender
             .declare(QueueSpecification.queue(QUEUE_NAME)
                 .durable(true)
-                .arguments(rabbitMQConfiguration.workQueueArgumentsBuilder(ALLOW_QUORUM)
+                .arguments(rabbitMQConfiguration.workQueueArgumentsBuilder(!AUTO_DELETE_QUEUE)
```

Comment:
Hmmmm... I really don't see it here... `workQueueArgumentsBuilder` takes a `allowQuorum` boolean argument, so I don't see what the auto delete here is doing.

Also for the queue to be quorum, it needs that argument to be true and to have quorum enabled in the conf. But here you put it to false... so even if quorum is enabled in rabbitmq.properties, it will never be quorum... 

Why are you doing trying this? Cause I don't get it sorry

Reply:
![image](https://github.com/apache/james-project/assets/6928740/51df987f-493f-449f-89fb-52a40dfd79b4)

on 3.7.x

Reply:
Fair point :)